### PR TITLE
V1.1.5

### PR DESCRIPTION
**Pull request by CodyAustinDavis**

### CODE REVIEW

File: src/main/java/liquibase/ext/databricks/database/DatabricksConnection.java [hunk 1]
```diff
@@ -144,7 +143,9 @@ public int getDatabaseMinorVersion() throws DatabaseException {
     /////////////////////////////////////////////////// copy from parent ///////////////////////////////////////////////////
     @Override
     protected String getConnectionUrl() throws SQLException {
-        return con.getMetaData().getURL();
+        String rawUrl = con.getMetaData().getURL();
+        String updatedUrl = rawUrl + "UserAgentEntry=Liquibase;EnableArrow=0";
```
Comment (filipelautert) [inline src/main/java/liquibase/ext/databricks/database/DatabricksConnection.java:147]: Could there be cases where you need to add a `;` at the beginning of this appended string - like if the user doesn't add this?

File: src/main/java/liquibase/ext/databricks/snapshot/jvm/SchemaSnapshotGeneratorDatabricks.java [hunk 2]
```diff
@@ -16,113 +17,35 @@
...
-            if (schemas != null) {
-                schemas.close();
-            }
-        }
-
-        return returnList.toArray(new String[returnList.size()]);
-    }
-
-    @Override
-    protected DatabaseObject snapshotObject(DatabaseObject example, DatabaseSnapshot snapshot) throws DatabaseException, InvalidExampleException {
```
Comment (filipelautert) [inline src/main/java/liquibase/ext/databricks/snapshot/jvm/SchemaSnapshotGeneratorDatabricks.java:57]: This method should be really gone?

**Review by filipelautert — CHANGES_REQUESTED** [review]

File: src/main/java/liquibase/ext/databricks/snapshot/jvm/SchemaSnapshotGeneratorDatabricks.java (hunk 2 above)
Comment (CodyAustinDavis) [reply src/main/java/liquibase/ext/databricks/snapshot/jvm/SchemaSnapshotGeneratorDatabricks.java:57]: Its an override, so its not gone, but this was the reason things were breaking, it wasnt able to find the schema and tables correctly when it did a snapshot - and returned an empty snapshot of table/column objects. Removing this custom override fixed it.

File: src/main/java/liquibase/ext/databricks/database/DatabricksConnection.java (hunk 1 above)
Comment (CodyAustinDavis) [reply src/main/java/liquibase/ext/databricks/database/DatabricksConnection.java:147]: Yea probably. Ill change this and make it check first. I think I should check for the before and after ";"

Comment (CodyAustinDavis) [reply src/main/java/liquibase/ext/databricks/database/DatabricksConnection.java]: Made this fix. Let me know what you think.

**Review by filipelautert — APPROVED** [review]

**Comment by filipelautert** [comment]
@tati-qalified this is ready to merge then release. Could you give it a try and see if it's working fine?

**Pull request merged**